state.go

```diff
@@ -92,78 +90,39 @@ func (s state) validate() (bool, string) {
 
 		if s.Settings.ClusterURI != "" && !s.Settings.BearerToken {
 			if !caCrt || !caKey {
-				return false, "ERROR: certificates validation failed -- You want me to connect to your cluster for you " +
+				return false, "certificates validation failed -- You want me to connect to your cluster for you " +
 					"but have not given me the cert/key to do so. Please add [caCrt] and [caKey] under Certifications. You might also need to provide [clientCrt]."
 			}
 
 		} else if s.Settings.ClusterURI != "" && s.Settings.BearerToken {
 			if !caCrt {
-				return false, "ERROR: certificates validation failed -- cluster connection with bearer token is enabled but " +
+				return false, "certificates validation failed -- cluster connection with bearer token is enabled but " +
 					"[caCrt] is missing. Please provide [caCrt] in the Certifications stanza."
 			}
 		}
 
 	} else {
 		if s.Settings.ClusterURI != "" {
-			return false, "ERROR: certificates validation failed -- kube context setup is required but no certificates stanza provided."
+			return false, "certificates validation failed -- kube context setup is required but no certificates stanza provided."
 		}
 	}
 
 	// namespaces
 	if nsOverride == "" {
 		if s.Namespaces == nil || len(s.Namespaces) == 0 {
-			return false, "ERROR: namespaces validation failed -- I need at least one namespace " +
+			return false, "namespaces validation failed -- I need at least one namespace " +
```

Comment:
Would remove `I need at least one namespace to work with!` for `at least one namespace is required.`

Reply:
:+1: